### ISSUE TITLE
Investigate visualizations

### ISSUE THREAD

**Issue by mikeswoods** [Andykmcc/project-graph#18]
This issue is for tracking ideas related to visualizing efforts and their relations.

**Comment by Andykmcc** [comment]
can you add links to the d3 stuff you found?

**Comment by mikeswoods** [comment]
@Andykmcc yeah. still working on it at the moment. I'll push the branch I'm playing around on and update this issue
'

**Comment by vincepmartin** [comment]
http://arborjs.org/
Found this while reading a stack overflow posting by one of the neo4j team. They seem quite eager to reach out to the community.

https://stackoverflow.com/questions/9175219/which-graph-database

**Comment by Andykmcc** [comment]
good find but sadly it is pretty old school. last commit was was 4-5 years ago. but i found this link in the issues. lots of options to investigate.

https://stackoverflow.com/questions/7034/graph-visualization-library-in-javascript

**Comment by Andykmcc** [comment]
this one seems like a really good option. it has one key maintainer with a good raod map ahead. we could contribute to it, help push through his road map
